corehq/apps/cachehq/cachemodels.py

```diff
@@ -76,7 +76,6 @@ class LocationGenerationCache(GenerationCache):
     generation_key = '#gen#location#'
     doc_types = ['Location']
     views = [
-        'commtrack/locations_by_code',
         '_all_docs',
```

Comment:
@dannyroberts do you know what `'_all_docs'` is doing here?  Can I just remove this whole class?